docs/README_API_INSTALL_DOCKER.md

```diff
@@ -16,34 +14,56 @@ This method requires [Docker](https://docs.docker.com/get-docker/) to be install
     cd WeVoteServer
     ```
 
-2. (Optional) Start WeVote Localstack
+2. Set environment variables in `.env`
 
-    Some parts of the WeVote API service utilize AWS services such as SQS. If you are developing this part of the WeVote API code, you can run a local AWS stack in a container to minimick real AWS services for testing. To start the localstack container, use the following command:
     ```
-    docker/dev_environment.sh localstack
+    # api
+    DATABASE_PASSWORD="secret"
+
+    # db
+    POSTGRES_PASSWORD="secret"
     ```
 
-3. Start development environment
+3. Configure PostgreSQL in `init.sql`
 
+    ```sql
+    ALTER SYSTEM SET listen_addresses = '*';
```

Comment:
Sure this is necessary? Using the dev_environment.sh script, they are started in the same docker namespaces and port 5432 (pgsql) is forwarded properly.

Reply:
According to the [image](https://hub.docker.com/_/postgres) under database configuration: `Important note: you must set listen_addresses = '*' so that other containers will be able to access postgres.`